spec/api/comment_spec.rb

```diff
@@ -41,12 +42,30 @@
 
       retrieved_children = retrieved['children']
       retrieved_children.length.should == comment.children.length
+      retrieved["child_count"].should == comment.children.length
 
       comment.children.each_with_index do |child, index|
         expect(retrieved_children[index]).to include('body' => child.body, 'parent_id' => comment.id.to_s)
       end
     end
 
+    it 'retrieve information of a single comment and fixes incorrect child count' do
```

Comment:
If you can also fix that above, where it was already set by someone else, that would be sweet.